tests/cases/parser/programStructure36.php

```diff
@@ -0,0 +1,5 @@
+<?php
+// FIXME the tolerant-php-parser has a bug, this code always echoes if executed.
+// (i.e. same as `if (false); echo "hello world"` with an implicit semicolon)
+// NOTE: If the inline HTML is surrounded by brackets, then this would never echo.
+if (false)?>hello world<?php
```

Comment:
This bug wasn't introduced by my PR, so I consider it out of scope. I filed https://github.com/Microsoft/tolerant-php-parser/issues/246 for followup work.

- The tree should change once the bug is fixed

See https://3v4l.org/ELbqZ